com.unity.render-pipelines.high-definition/Runtime/Material/LayeredLit/LayeredLit.shader

```diff
@@ -277,8 +277,8 @@ Shader "HDRP/LayeredLit"
         [HideInInspector] _StencilWriteMaskMV("_StencilWriteMaskMV", Int) = 32 // StencilUsage.ObjectMotionVector
 
         // Blending state
-        [HideInInspector] _SurfaceType("__surfacetype", Float) = 0.0
-        [HideInInspector] _BlendMode ("__blendmode", Float) = 0.0
+        _SurfaceType("__surfacetype", Float) = 0.0
```

Comment:
note: HideInInspector shouldn't matter here as we have our custom editor anyway

Reply:
actually, all the HideInInspector are useless because we have our own editor. But the tag will be used for material variants to display the number of overriden properties (it's a UX request). And we don't want to count these properties in